database/migrations/2022_01_05_100650_create_ayr_user_profiles_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAyrUserProfilesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('ayr_user_profiles', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('title');
+            $table->string('ref_id');
+            $table->string('profile_key');
+            $table->string('authorized_on');
```

Comment:
Use DateTime or timestamp field

Reply:
Changes Done will it work with -> Carbon::now()->toDateTimeString()